src/qililab/instruments/quantum_machines/quantum_machines_cluster.py

```diff
@@ -378,12 +378,39 @@ def get_parameter_of_bus(self, bus: str, parameter: Parameter):
             ParameterNotFound: Raised if parameter does not exist
         """
         # TODO: Change private QM API to public when implemented.
+
+        if "_config" not in dir(self):
```

Comment:
I don't think this is the place to solve this of course, but maybe we can decide if we want the `settings` always ruling over the `_config` in this case, just in case, removing the `conditional if` @fedonman??
```suggestion
```
(also we would need to untab next line...)

-----

Or else, if we want the `_config` to rule, for which then its good as it is

Reply:
Depending on what we decide here, that would also affect, how I write it in this other PR, for the `set_parameters` method: https://github.com/qilimanjaro-tech/qililab/pull/752

Reply:
In the `inital_setup` we are directly doing: `self._config = self.settings.to_qua_config()` like the settings rules over the `._config`, maybe we should follow that everywhere? 🤔

Reply:
# AFTER A MEETING, WE DECIDED:

1.a)  Synching or Prioritizing  settings vs _config (#754, or define sync?)
- :white_check_mark: Both are equally important/priorty - So we have to be careful with synching between them

1.b) Do we want to save the changed info in append_configuration or is it just for the execution?
- :white_check_mark:  We need it yes, no temporary!
-- :x: If not -> This would imply we can delete _config as an attribute, only property (at least not now)
-- :x: If we want -> Then we can do reverse to_qua_config() function
-- :white_check_mark: We then have to correctly define and check that both infos are synch all the time:

2.) Initialization of _qm and _qmm is kind of mixed, no? (setup needed before turn_on..)
- :white_check_mark: Move everything into initial_setup , but lets wait into a better moment? (to check/test properly)

Reply:
I'm going to commit some changes to adapt for these decisions 👍
Resolving this comment then